src/foundry/client-esm/applications/api/application.d.mts

```diff
@@ -182,11 +210,6 @@ declare namespace ApplicationV2 {
     controls: boolean;
   }
 
-  /**
-   * Context data provided to the renderer
-   */
-  export interface RenderContext extends Record<string, unknown> {}
```

Comment:
I decided to remove this. If a user declaration merges into `ApplicationV2.RenderContext` then it's borderline useless as subclasses must override it. If their intent is to indicate that all `ApplicationV2` instances need to be passed some option this is almost certainly unsound as Foundry hardcodes certain render context configs among different subclasses.

If they want this because of an unforseen Foundry update it then of course we can add this back (probably with merging into the subclass `RenderContext` not implemented even before this diff) or they can approach this from a perspective of the subclasses.